ivadomed/transforms.py

```diff
@@ -118,6 +118,8 @@ def __init__(self, dict_transforms, requires_undo=False):
 
             # call transform
             if transform in globals():
+                if transform == "NumpyToTensor":
```

Comment:
note this line fixes a bug for a merged commit https://github.com/ivadomed/ivadomed/pull/818, this line needs to go in asap